legend-engine-pure-code-compiled-core-configuration/src/main/resources/core_configuration/coreExtensions.pure

```diff
@@ -83,7 +83,8 @@ function <<LocalPlatformBinding.TestPlanBinder>> meta::pure::extension::configur
    (
       overrides = [
          meta::external::format::flatdata::executionPlan::platformBinding::legendJava::bindFlatdataTestPlanToPlatform__LocalPlatformBinder_1_,
-         meta::relational::executionPlan::platformBinding::legendJava::bindRelationalTestPlanToPlatform__LocalPlatformBinder_1_
+         meta::relational::executionPlan::platformBinding::legendJava::bindRelationalTestPlanToPlatform__LocalPlatformBinder_1_,
```

Comment:
this was the exact change that we reverted in one of our previous PR.  adding mongo to pure-code-compiled-core-configuration.  Guess this is the binder fix, but let us talk Rafael & Hardik on this.